Use ar to link static libraries to support universal builds.
I don't understand why upstream stopped using ar back in version 0.0.3.
--- build/bottom.make.orig	2016-02-25 12:41:01.000000000 -0600
+++ build/bottom.make	2019-01-08 18:50:57.000000000 -0600
@@ -66,10 +66,7 @@
 
 $($(X_MODULE)_OUTPUT)/$(BINARY)$(X_LIBEXT): $($(X_MODULE)_OBJS)
 	@echo '  LINK LIB  $@'
-	$(Q)$(LD) -r -o '$@' $^
-#	# no AR anymore due to static initilizers
-#	$(Q)$(AR) r '$@' $^ 2> /dev/null
-#	$(Q)$(RANLIB) '$@'
+	$(Q)$(AR) -rcs '$@' $^
 
 $($(X_MODULE)_OUTPUT)/$(BINARY)$(X_DYNEXT): $($(X_MODULE)_OBJS)
 	@echo '  LINK DYN  $@'
